### ISSUE TITLE
How do I locate saved screenshots?

### ISSUE THREAD

**Issue by im-coder-lg** [weiameili/Pensela#21]
Hi,

Sorry for the template, GitHub Mobile didn't allow me to use issue forms.

I have tried taking numerous screenshots and checked my Pictures directory, my Documents directory but I can not happen to locate my screenshots. Can I know where screenshots are saved in Windows, macOS and Linux? That might help me if I happen to install Pensela on other OSes.

**Comment by weiameili** [comment]
They should be in a folder by the name "Pensela" in your system's default Pictures directory

**Comment by im-coder-lg** [comment]
Yeah the directory does exist but the picture doesn't... Potential bug? Or am I using an old version? I will try getting the latest version and try again.

**Comment by im-coder-lg** [comment]
Confirmed bug as of version 1.2.4 so let's start finding fixes soon!

**Comment by im-coder-lg** [comment]
Info: I think it has to do with a screen capture program on the unzipped version(used unzipped for skipping installation). I think it's a 32-bit program, check the unzipped version later @weiameili

**Comment by im-coder-lg** [comment]
Yep, found the bug.

**Comment by im-coder-lg** [comment]
I found the files here: `C:\Users\%USERPROFILE%\Downloads\Pensela-1.2.4-win\resources\app.asar.unpacked\node_modules\screenshot-desktop\lib\win32` and found the files that run as 32-bit files. And, as I expected, they were the Screen Capture files. We need to fix this bug soon.

**Comment by im-coder-lg** [comment]
/ping @weiameili any ideas? Or is this a genuine bug?

**Comment by weiameili** [comment]
The problem is that the files belong to the "screenshot-desktop" npm package that this app uses, so there isn't much we can do about it. The logical next step in solving this would be opening an issue on the package's repo, but for that, we should preferably reproduce this issue in a node script using just the screenshot-desktop package.

**Comment by im-coder-lg** [comment]
Well, we could try that, right?

**Comment by weiameili** [comment]
Well yes, but right now, I'm unable to reproduce this bug even using Pensela.

**Comment by weiameili** [comment]
Also, could you possibly build pensela yourself and see if the issue persists? (I had issues with the Pensela executable for Windows built in Linux, but when I built it in windows again, it was working alright)

**Comment by ldmpub** [comment]
Hi,

Bug confirmed running `Pensela 1.2.5` under **Windows 10 64bits** both with "portable" (zip) version and "installed" (setup.exe) version: 
* ✔a new Pensela folder is created (under the user>Images folder)
* ❌but screenshot images are not save here ... (and nowhere else in my computer as far as I can see 😕).

![image](https://user-images.githubusercontent.com/1452146/170862948-bd58b250-b0c8-4ddc-9d52-3547e34099ba.png)

PS : screenshot above has been taken with an other screenshot tool because Pensela can not take / save any screenshot on my computer 😜.
